swagger-current.yml

```diff
@@ -957,12 +957,62 @@ components:
                 - CE
                 - SPG
               description: case type
+            uprn:
+              type: string
+              pattern: '^\d{1,12}$'
+              description: The address UPRN if known, or null if not known.
+            estabType:
```

Comment:
as discussed , please see if we can componentise these common enums. particularly estabType since it is a lengthy list.